docs/content/content/types.md

```diff
@@ -60,16 +60,15 @@ Create a directory with the name of the type in `/layouts`. Type is always singu
 ### Create single template
 Create a file called `single.html` inside your directory. *E.g. `/layouts/post/single.html`*.
 
-### Create list template
-Create a file called `list.html` inside your directory. *E.g. `/layouts/post/list.html`*.
-
 ### Create views
-Many sites support rendering content in a few different ways, for instance,
-a single page view and a summary view to be used when displaying a list
-of contents on a single page. Hugo makes no assumptions here about how you want
-to display your content, and will support as many different views of a content
-type as your site requires. All that is required for these additional views is
-that a template exists in each `/layouts/TYPE` directory with the same name.
+Many sites support rendering content in a few different ways, for
```

Comment:
That's interesting.  No, it's not related.  That commit was already merged to master.  I'm almost positive that commit wasn't in my PR when I submitted it.  Not sure what happened.  I've rebased and pushed.  Only one commit listed now.